code/frontend/src/app/features/settings/download-cleaner/download-cleaner.component.html

```
@@ -140,6 +140,9 @@ <h4 class="rule-card__name">{{ rule.name }}</h4>
                         @if (rule.minSeedTime > 0) {
                           <app-badge>Min Seed: {{ rule.minSeedTime }}h</app-badge>
                         }
+                        @if ((rule.minSeeders ?? 0) > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)